jitsi-utils/src/test/java/org/jitsi/utils/queue/DummyQueue.java

```diff
@@ -31,13 +31,13 @@
         PacketHandler<Dummy> packetHandler,
```

Comment:
We have 2 `DummyQueue`s now?

Reply:
One in jitsi-utils and one in jitsi-utils-kotlin, since maven package dependencies don't import their dependencies' test directories.